include/tvm/relay/module.h

```diff
@@ -104,18 +104,20 @@ class ModuleNode : public RelayNode {
    * \param update Controls whether you can replace a definition in the
    * environment.
    */
-  TVM_DLL void AddDef(const GlobalTypeVar& var, const TypeData& type, bool update = false);
+  TVM_DLL void AddTypeDef(const GlobalTypeVar& var, const TypeData& type, bool update = false);
 
   /*!
-   * \brief Add a type definition to the global environment.
-   * \param var The name of the global function.
+   * \brief Add a type-level definition to the global environment.
+   * \param var The var of the global type definition.
    * \param type The ADT.
    * \param update Controls whether you can replace a definition in the
    * environment.
    *
-   * It does not do type inference as AddDef does.
+   * It does not do kind checking as AddTypeDef does.
```

Comment:
```suggestion
   * It does not do type checking as AddTypeDef does.
```

`kind` is [type of type](https://en.wikipedia.org/wiki/Kind_(type_theory)), here should be type checking instead.